pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -186,10 +189,33 @@ func (l *LocalTaskRefResolver) GetTask(ctx context.Context, name string) (v1beta
 	if l.Namespace == "" {
 		return nil, fmt.Errorf("must specify namespace to resolve reference to task %s", name)
 	}
-	return l.Tektonclient.TektonV1beta1().Tasks(l.Namespace).Get(ctx, name, metav1.GetOptions{})
+	task, err := l.Tektonclient.TektonV1beta1().Tasks(l.Namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	if err := verifyResolvedTask(ctx, task.DeepCopy()); err != nil {
```

Comment:
same comment as above, is it possible to centralize the two usages to just one place?